mkroosevelt.js

```diff
@@ -45,7 +51,14 @@ else if (cmd === '-v' || cmd === '--v' || cmd === '-version' || cmd === '--versi
 }
 else if (cmd) {
   try {
-    createSampleApp();
+    if (cmd === '.') {
+      currentDirectory = true;
+      createSampleApp(currentDirectory);
```

Comment:
Can you check to see if this will overwrite any existing files in the directory first and if so warn the user that it will overwrite them and ask for confirmation (y/N)?